docs/source/en/core/development.md

```diff
@@ -1 +1,339 @@
-this document is still waiting for translation, see [Chinese Version](/zh-cn/core/development.html)
\ No newline at end of file
+title: Local Development
+---
+
+To improve development experience, convenient ways to develop in local environment, debug and run unit testing are provided in egg. 
+
+Here, wo will need to use [egg-bin] module (Only used in local development and unit testing. For production environment, please refer to [Deployment](./deployment.md)).
+
+First of all, we need to include `egg-bin` module in `devDependencies`:
+
+```bash
+$ npm i egg-bin --save-dev
+```
+
+## Start App
+
+Once we have modified code and saved in local development, the app will restart automatically, and our changes will take place right after that. 
+
+### Add Script
+
+Add `npm scripts` into `package.json`：
+
+```json
+{
+  "scripts": {
+    "dev": "egg-bin dev"
+  }
+}
+```
+
+And then we may start app by command `npm run dev`.
+
+### Config Environment
+
+To start app in local, environment need to be set as `env: local`. The configuration being read is an object combined from `config.local.js` and `config.default.js` files.
```

Comment:
1) need => needs.

2) The configuration comes from the combination of both `config.local.js` and `config.default.js`.  (No need to translate word by word but just its main meaning to fit for the English manner)